README.md

```diff
@@ -38,10 +41,11 @@ We recommend using [`buf-setup-action`][buf-setup] to install it (as in the exam
 
 ## Configuration
 
-Parameter | Description | Required | Default
-:---------|:------------|:---------|:-------
-`buf_token` | The [Buf authentication token][buf-token] used for private [Buf inputs][input] | ✅  | [`${{github.token}}`][github-token]
-`input` | The path of the [input] you want to push to BSR as a module | | `.`
+| Parameter      | Description                                                                                           | Required | Default                             |
+|:---------------|:------------------------------------------------------------------------------------------------------|:---------|:------------------------------------|
+| `buf_token`    | The [Buf authentication token][buf-token] used for private [Buf inputs][input]                        | ✅        | [`${{github.token}}`][github-token] |
+| `input`        | The path of the [input] you want to push to BSR as a module                                           |          | `.`                                 |
+| `enable_draft` | Enable to push the BSR commit as draft when the git branch that triggering the workflow is not `main` |          |                                     |
```

Comment:
```suggestion
| `draft` | Indicates if non-`main` workflows should be ignored or push to the BSR as draft commit  |          |                                     |
```
With the param being a boolean, I don't think we need a `enable_` prefix? Do we do that in other github actions?

If we keep this as just `drafts`, we can later extend it to check if its a boolean or a string, and in the string case only push if it matches.